cmd/internal/find/primary/meta.go

```diff
@@ -82,3 +86,440 @@ var metaPrimary = Parser.add(&primary{
         }
     },
 })
+
+const metaDetailedDescription = `
```

Comment:
This is pretty long. Open to ideas on how to make it shorter/reorganize it/whatever-else-it-takes to make all of meta's features more explicit.